pkg/middleware/dashboard_redirect_test.go

```diff
@@ -55,4 +55,21 @@ func TestMiddlewareDashboardRedirect(t *testing.T) {
 			})
 		})
 	})
+
+	Convey("Given the dashboard legacy edit panel middleware", t, func() {
```

Comment:
We might want to rewrite this using the standard testing library.

Reply:
Then we have to convert this whole file, what test runners are you using?

Reply:
@torkelo We just use `go test` for running tests. In this case, I think it would be easy to write this test without Convey. That's what I usually do when adding new tests, just write them with the standard library, in addition to the existing Convey ones (which I guess we'll clean up at some point).

Reply:
go test is terrible, have scroll a bunch to see what failed due to stack traces that are more than a screen long, and it does not rerun tests on file change (no watch), so the whole experience I find very poor. Like having a test runner in a tmux split that runs tests on file change and print failures in readable form. 

I tried to rewrite this to go testing library but sadly it depends on middlewareScenario which uses GoConvey so have to rewrite hundreds of tests to go testing library to convert this test :(